README.md

```
@@ -117,44 +125,44 @@ This repository only contains the source code for the package.
 1. Publish the generated artifacts to the local Ballerina central repository:
-   ```
+   ```shell
    ./gradlew clean build -PpublishToLocalCentral=true
    ```
 1. Publish the generated artifacts to the Ballerina central repository:
-   ```
+   ```shell
    ./gradlew clean build -PpublishToCentral=true
    ```      
 
 ## Contribute to Ballerina
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)